### PR TITLE
app checkin changes nodejs

### PR DESCRIPTION
**Pull request by AbhijeetThakur-msft**

### CODE REVIEW

File: samples/app-checkin-location/nodejs/Readme.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+## Get Checkin info of user
+
```
Comment (v-chetsh) [inline samples/app-checkin-location/nodejs/Readme.md:2]: Kindly make sure to link this file with root readme file.

File: samples/app-checkin-location/nodejs/bots/teamsBot.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        this.onMessage(async (context, next) => {
+            if (context.activity.text.toLowerCase() == "viewcheckin") {
+                const fileJsonString = fs.readFileSync("../nodejs/public/checkindetails.json", "utf8");
+                let prevCheckinDetails = new Array();
+
+                // Check if file is empty.
+                if (fileJsonString == "") {
+                    await context.sendActivity("No last check in found");
+                }
+
```
Comment (v-chetsh) [inline samples/app-checkin-location/nodejs/bots/teamsBot.js:34]: Remove extra empty line

File: samples/app-checkin-location/nodejs/bots/teamsBot.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+                    allCheckinDetails.find((user) => {
+                        if (user.userId == context.activity.from.aadObjectId) {
+                            prevCheckinDetails.push(user);
+                        }
+                    });
+
+                    if (prevCheckinDetails.length == 0) {
+                        await context.sendActivity("No last check in found");
+                    }
+
```
Comment (v-chetsh) [inline samples/app-checkin-location/nodejs/bots/teamsBot.js:47]: Remove extra empty line

File: samples/app-checkin-location/nodejs/bots/teamsBot.js [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+                        prevCheckinDetails.map((user) => {
+                            let userCheckinCard = CardFactory.adaptiveCard(this.adaptiveCardForUserLastCheckin(user));
+                            cardArray.push(userCheckinCard);
+                        })
+
+                        await context.sendActivity({ attachments: cardArray });
+                    }
+                }
+            }
+
```
Comment (v-chetsh) [inline samples/app-checkin-location/nodejs/bots/teamsBot.js:60]: Remove extra empty line

File: samples/app-checkin-location/nodejs/bots/teamsBot.js [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+    // This method is used to save user's checkin details to file.
+    saveUserDetails = (newCheckinDetails) => {
+        var fileJsonString = fs.readFileSync("../nodejs/public/checkindetails.json", 'utf8');
+
+        // Check if file is empty.
+        if (fileJsonString == "") {
+            fs.writeFileSync("../nodejs/public/checkindetails.json", JSON.stringify([newCheckinDetails]), "utf-8");
+        }
+
```
Comment (v-chetsh) [inline samples/app-checkin-location/nodejs/bots/teamsBot.js:118]: Remove extra empty line

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**